app/models/registration_holder.rb

```diff
@@ -13,24 +13,43 @@ def assign_courses
     self.sixth_course = courses[5] if courses[5]
   end
 
+  def amount_owed
+    course_count = 0
+    course_count += 1 if self.first_role != "None"
+    course_count += 1 if self.second_role != "None"
+    course_count += 1 if self.third_role != "None"
+    course_count += 1 if self.fourth_role != "None"
+    if course_count > 1
+      return 9000
+    elsif course_count == 1
+      return 5000
+    else
+      return "Error!"
+    end
+  end
+
   def create_each_registration
     assign_courses
     registrations = []
     registrations << CourseRegistration.new(student_id: self.student_id,
                                      course_id: self.first_course,
-                                     role: self.first_role)
+                                     role: self.first_role,
+                                     paid: true)
 
     registrations << CourseRegistration.new(student_id: self.student_id,
                                       course_id: self.second_course,
-                                      role: self.second_role)
+                                      role: self.second_role,
+                                      paid: true)
 
     registrations << CourseRegistration.new(student_id: self.student_id,
                                       course_id: self.third_course,
-                                      role: self.third_role)
+                                      role: self.third_role,
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Redundant `self` detected.